examples/lib/stories/controls/controls.dart

```diff
@@ -18,61 +19,62 @@ void addControlsStories(Dashbook dashbook) {
     ..add(
       'Keyboard',
       (_) => GameWidget(game: KeyboardGame()),
-      codeLink: baseLink('gestures/keyboard.dart'),
+      codeLink: baseLink('controls/keyboard.dart'),
     )
     ..add(
       'Mouse Movement',
       (_) => GameWidget(game: MouseMovementGame()),
-      codeLink: baseLink('gestures/mouse_movement.dart'),
+      codeLink: baseLink('controls/mouse_movement.dart'),
     )
     ..add(
       'Scroll',
       (_) => GameWidget(game: ScrollGame()),
-      codeLink: baseLink('gestures/scroll.dart'),
+      codeLink: baseLink('controls/scroll.dart'),
     )
     ..add(
       'Multitap',
       (_) => GameWidget(game: MultitapGame()),
-      codeLink: baseLink('gestures/multitap.dart'),
+      codeLink: baseLink('controls/multitap.dart'),
     )
     ..add(
       'Multitap Advanced',
       (_) => GameWidget(game: MultitapAdvancedGame()),
-      codeLink: baseLink('gestures/multitap_advanced.dart'),
+      codeLink: baseLink('controls/multitap_advanced.dart'),
     )
     ..add(
       'Tapables',
       (_) => GameWidget(game: TapablesGame()),
-      codeLink: baseLink('gestures/tappables.dart'),
+      codeLink: baseLink('controls/tappables.dart'),
     )
     ..add(
       'Overlaping Tappables',
       (_) => GameWidget(game: OverlappingTapablesGame()),
-      codeLink: baseLink('gestures/overlaping_tappables.dart'),
+      codeLink: baseLink('controls/overlaping_tappables.dart'),
     )
     ..add(
       'Draggables',
       (context) {
         return GameWidget(
           game: DraggablesGame(
-            zoom: context.listProperty(
-              'zoom',
-              1,
-              [0.5, 1, 1.5],
-            ),
+            zoom: context.listProperty('zoom', 1, [0.5, 1, 1.5]),
           ),
         );
       },
-      codeLink: baseLink('gestures/draggables.dart'),
+      codeLink: baseLink('controls/draggables.dart'),
+    )
+    ..add(
+      'Hoverables',
+      (_) => GameWidget(game: HoverablesGame()),
+      codeLink: baseLink('controls/hoverables.dart'),
```

Comment:
Add some info about this example, could be just some instructions on how the example works.